eap-xp3/image.yaml

```diff
@@ -0,0 +1,138 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp3-openjdk11-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform XP 3.0 OpenShift container image."
+version: "3.0"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-7-xp3-openjdk11-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP XP 3.0"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP XP"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "builder,javaee,eap,eap7,eap-xp"
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+ports:
+    - value: 8443
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: 0.45.2
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/luck3y/jboss-eap-modules.git
```

Comment:
this should point to the https://github.com/jboss-openshift org.

Reply:
Ooops sorry pushed by acciedent.